app/views/coronavirus_landing_page/components/shared/_section.html.erb

```diff
@@ -5,7 +5,23 @@
   <ul class="covid__list covid__list--accordion">
     <% sub_section["list"].each do | item | %>
       <li class="covid__list-item">
-        <a class="govuk-link covid__accordion-link <%= "covid__page-guidance-link" if item["featured_link"] %>" href="<%= item["url"] %>"><%= item["label"] %></a>
+        <% if item["featured_link"] %> 
+          <%= 
+            render 'components/action-link', {
```

Comment:
Nitpick: our convention is normally to include these two lines on one line, i.e.

`<%= render 'components/action-link', {`

Similarly for the closing line, should be

`} %>`

Reply:
Good to know! I've re-formatted this